### DISCUSSION TITLE
Hexagonal pitch Modelling in Moltres

### DISCUSSION THREAD

**Discussion by saqibhayat556** [arfc/moltres#246] [Q&A Reactor multiphysics] [answered]
Hi @smpark7 Sir! My question is how to model my geometry with hexagonal pitch in 2D along RZ? My geometry, unlike MSRE, is not same in 'xz' and 'yz' plane. Below images are attached for my geometry from openmc. I want to analyze this geometry like MSRE example but don't know whether i am modelling this geometry right in 'Gmsh' or not. I am modelling my geometry in 'xz' plane in 'Gmsh' because of pitch information along x-axis. But i am not getting the right results in Moltres for this geometry.

![top_view_xy](https://github.com/arfc/moltres/assets/124503670/f04b5260-8299-4949-a4e0-abb1ff144bd4)
![xz](https://github.com/arfc/moltres/assets/124503670/79ed0e90-0836-4cde-bdc4-1db048c5395c)
![yz](https://github.com/arfc/moltres/assets/124503670/e88c7186-33d7-4ff6-a0ed-a5b1dabd801f)

**Comment by smpark7** [comment, marked as answer]
Hey @saqibhayat556. I think the equivalent 2D RZ model of this reactor, which I assume is Seaborg's design, will be somewhere in between the XZ and YZ models. The RZ model is equivalent to a 3D model consisting of alternating cylindrical shells of fuel and moderator. You have to tweak the RZ model by trial and error to obtain similar neutronic properties between the original 3D model and the representative RZ model.

**Reply by saqibhayat556** [reply]
Thank you!
